JASP-Engine/JASP/po/R-JASP.pot

```diff
@@ -622,7 +622,7 @@ msgstr ""
 msgid "Model fit is singular. Specified random effects parameters (random intercepts and random slopes) cannot be estimated from the available data. Carefully reduce the random effects structure, but this practice might inflate the reported p-value, and invalidates the analysis."
 msgstr ""
 
-msgid "Vovk-Sellke Maximum <em>p</em>-Ratio: Based on a two-sided <em>p</em>-value, the maximum possible odds in favor of H%1$s over H%2$s equals 1/(-e <em>p</em> log(<em>p</em>)) for <em>p</em> %3$s .37 (Sellke, Bayarri, & Berger, 2001)."
+msgid "Vovk-Sellke Maximum <em>p</em>-Ratio: Based on a two-sided <em>p</em>-value, the maximum possible odds in favor of H1 over H0 equals 1/(-e <em>p</em> log(<em>p</em>)) for <em>p</em> = .37 (Sellke, Bayarri, & Berger, 2001)."
```

Comment:
Did you forget to add this change in  MixedModelsCommon.R in your commit? How such a change can come in the pot file, if the original string is not changed?

Reply:
Someone else changed it in `stable` I think, it appeared when I ran the "language generation" thing of jasp.
But maybe I made a mistake?

Reply:
Hmm, I can't see where it came from to be honest. I'll revert that.